docs-kits_versioned_docs/version-23.09/kits/tractusx-edc/docs/samples/management-api-v2-walkthrough/2-assets.md

```diff
@@ -1,6 +1,6 @@
 # Creating an Asset
```

Comment:
please remove the whole `management-api-v2-walkthrough` folder as it already exists in the `tractusx-edc` repo

Reply:
Please don't forget … the folder vesrion-23.09 represents the release documentation. The current folder "work in progress" is https://github.com/eclipse-tractusx/eclipse-tractusx.github.io/tree/main/docs-kits ...

Reply:
didn't notice sorry, so my question is why modify such files in this PR? I would exclude them (eventually address in a different PR)

Reply:
I reverted the changes - thanks for pointing that out.

After [mgmt-api walkthrough has been merged in tractusx-edc](https://github.com/eclipse-tractusx/tractusx-edc/pull/953), I'd now consider this PR final.

What's left then is only to [remove this kit-section in tractusx-edc](https://github.com/eclipse-tractusx/tractusx-edc/issues/980).